pkg/ccl/backupccl/full_cluster_backup_restore_test.go

```diff
@@ -92,6 +92,7 @@ func TestFullClusterBackup(t *testing.T) {
 	// Populate system.users.
 	for i := 0; i < 1000; i++ {
 		sqlDB.Exec(t, fmt.Sprintf("CREATE USER maxroach%d", i))
+		sqlDB.Exec(t, fmt.Sprintf("ALTER USER maxroach%d CREATEDB", i))
```

Comment:
@solongordon this is a 20.2 role, right?

Reply:
Updated.